src/debugpy/_vendored/pydevd/_pydevd_sys_monitoring/_pydevd_sys_monitoring.py

```diff
@@ -624,10 +624,10 @@ def _get_func_code_info(code_obj, frame_or_depth) -> FuncCodeInfo:
             frame = frame_or_depth
         assert frame.f_code is code_obj
 
-    func_code_info.filtered_out_force_checked = py_db.apply_files_filter(frame, func_code_info.abs_path_filename, py_db.get_use_libraries_filter())
+    func_code_info.filtered_out_force_checked = py_db.apply_files_filter(frame, func_code_info.abs_path_filename, True)
```

Comment:
This needs to be True as this flag is used elsewhere. However `always_filtered_out` should be determined by whether or not the file is filtered (which it isn't when justMyCode = false)